androidApp/src/androidTest/java/com/mbta/tid/mbta_app/android/TestUtils.kt

```diff
@@ -1,9 +1,57 @@
 package com.mbta.tid.mbta_app.android
 
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toPixelMap
 import androidx.compose.ui.semantics.SemanticsActions
 import androidx.compose.ui.test.SemanticsMatcher
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.captureToImage
+import kotlin.math.abs
+import org.junit.Assert.fail
 
 fun hasClickActionLabel(expected: String?) =
     SemanticsMatcher("has click action label $expected") { node ->
         node.config[SemanticsActions.OnClick].label == expected
     }
+
+@OptIn(ExperimentalStdlibApi::class)
+private fun colorToHex(color: Color): String {
+    fun valToHex(value: Float): String {
+        val byteValue = (value * 255).toInt().toByte()
+        return byteValue.toHexString(HexFormat.UpperCase)
+    }
+    return "#${valToHex(color.red)}${valToHex(color.green)}${valToHex(color.blue)}"
+}
+
+private fun Color.isGrey(): Boolean =
+    abs(this.red - this.green) < 0.01 && abs(this.green - this.blue) < 0.01
+
+/**
+ * It's difficult to determine if a view hierarchy contains the correct image when the image is
+ * semantically invisible, but it's easy to capture a screenshot of the view and check it pixel by
+ * pixel.
+ */
+fun SemanticsNodeInteraction.assertHasColor(targetColor: Color) {
+    val pixelMap = this.captureToImage().toPixelMap()
```

Comment:
I knew there was a chance this existed. I may have been too eager to use it, but I'm tired of setting tags on things.